.travis/travis-ci.sh

```diff
@@ -1,5 +1,23 @@
 #!/bin/bash
 
+# if requested, run all tests
+if [[ "$TRAVIS_COMMIT_MESSAGE" != *"[force TravisCI]"* ]]; then
```

Comment:
I dislike us putting stuff like this in commit messages that remain part of our git history. They're irrelevant to history. Is there any way to have this controlled via pull request title or description rather than via commit message?

Reply:
No. What can be done is check a pattern in the name of the branch from which the PR originated.

Reply:
I think "check a pattern in the name of the branch from which the PR originated" is preferable. Can you make it e.g. "force-travis"? I assume it'd apply to branch names in other GitHub accounts' repos, as long as those are used to initiate the PR from, right? Thanks!

Reply:
Also, did you test this script (with the proposed changes) somehow, perhaps setting the env vars on your computer?

Reply:
Travis.org still works.